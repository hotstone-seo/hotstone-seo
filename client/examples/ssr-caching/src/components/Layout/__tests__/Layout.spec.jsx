import React from 'react';
import Layout from '../index.js';

test('Should render Layout correctly', () => {
  const tree = mount(<Layout route={{ routes: [] }}/>);
  
  expect(tree).toMatchSnapshot();
});
