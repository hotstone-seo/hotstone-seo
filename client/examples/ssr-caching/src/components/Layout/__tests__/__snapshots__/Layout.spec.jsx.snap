// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Layout correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Router
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "?order_id=39982221&order_hash=2a71d91259eefafd4bea3465050d4fa7ecb4186e",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <Layout
        route={
          Object {
            "routes": Array [],
          }
        }
      >
        <div>
          <Switch />
        </div>
      </Layout>
    </Router>
  </BrowserRouter>
</Provider>
`;
